ivy/data_classes/array/experimental/norms.py

```diff
@@ -39,93 +39,123 @@ def l2_normalize(
         """
         return ivy.l2_normalize(self, axis=axis, out=out)
 
+    def batch_norm(
+        self,
+        mean: Union[ivy.NativeArray, ivy.Array],
+        variance: Union[ivy.NativeArray, ivy.Array],
+        /,
+        *,
+        offset: Optional[Union[ivy.NativeArray, ivy.Array]] = None,
+        scale: Optional[Union[ivy.NativeArray, ivy.Array]] = None,
+        training: bool = False,
+        eps: float = 1e-5,
+        momentum: float = 1e-1,
+        out: Optional[ivy.Array] = None,
+    ) -> ivy.Array:
```

Comment:
And you haven't updated the type-hints for the return of the functions.

Reply:
Updated :+1:

Reply:
Since the function returns the normalised input, the mean and the variance, shouldn't the return type-hint be a tuple of ivy arrays instead of a single ivy array?

Reply:
Had missed this, have updated it now.